src/buttons/radio.ts

```diff
@@ -90,6 +90,11 @@ export class NgbRadio implements OnDestroy {
     }
   }
 
+  @Input('checked')
+  set checked(value: any) {
+    this._checked = true;
```

Comment:
Wouldn't this fail for ` <input type="radio" name="State" [checked]="false" />`? Could you please add this case to the unit test?